bin/perseo

```diff
@@ -81,9 +82,11 @@ function loadConfiguration() {
     if (process.env.PERSEO_ENDPOINT_PORT) {
         config.endpoint.port = process.env.PERSEO_ENDPOINT_PORT;
     }
-    if (process.env.PERSEO_MONGO_HOST) {
-        config.mongo.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/cep';
-        config.orionDb.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/orion';
+    if (process.env.PERSEO_MONGO_ENDPOINT) {
```

Comment:
I mean PERSEO_MONGO_ENDPOINT as a list of endpoints ([host:port], not a mongodb connection string